cli/templateversions.go

```diff
@@ -1,15 +1,80 @@
 package cli
 
-import "github.com/spf13/cobra"
+import (
+	"fmt"
+
+	"github.com/jedib0t/go-pretty/v6/table"
+	"github.com/spf13/cobra"
+	"golang.org/x/xerrors"
+
+	"github.com/coder/coder/cli/cliui"
+	"github.com/coder/coder/codersdk"
+)
 
 func templateVersions() *cobra.Command {
-	return &cobra.Command{
+	cmd := &cobra.Command{
 		Use:     "versions",
+		Short:   "Manage different versions of the specified template",
 		Aliases: []string{"version"},
+		Example: formatExamples(
+			example{
+				Description: "Show versions of a specific template",
+				Command:     "coder templates versions show my-template",
+			},
+		),
+	}
+	cmd.AddCommand(
+		templateVersionsShow(),
+	)
+
+	return cmd
+}
+
+func templateVersionsShow() *cobra.Command {
+	return &cobra.Command{
+		Use:   "show <template>",
```

Comment:
This should probably be `list` instead of `show` since it is a listing. `show` should be reserved for showing details about a single object.

Reply:
Yes, that sounds perfect! Thank you.